src/voter.py

```diff
@@ -16,7 +19,7 @@ def __init__(self, voter_id: int, M: int, vote: Optional[int] = None):
 
     def do_vote(self, tablet: Tablet):
         if self.vote is None or self.vote >= self.M:
-            self.vote = random.choice(range(self.M))
+            self.vote = random.choice(range(DEFAULT_NUM_CANDIDATES))
```

Comment:
Why is `M` no longer the range of possible vote values?

Reply:
It is, I just wanted to simulate a more realistic scenario, since M is supposed to be large. Can change this back if you want

Reply:
Understood. Keeping like this is fine by me.